test/fixtures/track-event-android.json

```diff
@@ -38,7 +38,7 @@
   },
   "output": {
     "appsflyer_id": "gucci-mane",
-    "advertisingId": "159358",
+    "advertising_id": "159358",
```

Comment:
this is what's sent to the appsflyer API, so we are matching on it exactly (instead of the case insensitivity). it wasn't matching before (but the assertion wasn't being run)